src/main/java/com/hellosign/sdk/HelloSignClient.java

```diff
@@ -746,6 +745,28 @@ public File getFiles(String requestId, String format) throws HelloSignException
         return httpClient.withAuth(auth).withGetParam(PARAM_FILE_TYPE_URI, format).get(url).asFile(fileName);
     }
 
+    public String getFilesUrl(String requestId) throws HelloSignException {
+        String fileUrl = null;
+        String url = BASE_URI + SIGNATURE_REQUEST_FILES_URI + "/" + requestId;
+
+        HttpClient httpClient = this.httpClient.withAuth(auth).withGetParam(PARAM_TEMPLATE_GET_URL, "1").get(url);
+
+        if (httpClient.getLastResponseCode() == 404) {
+            throw new HelloSignException(String.format("Could not find request with id=%s", requestId));
+        }
+
+        JSONObject response = httpClient.asJson();
+
+        if (response.has("file_url")) {
+            try {
+                fileUrl = response.getString("file_url");
+            } catch (JSONException ex) {
+                throw new HelloSignException(ex);
+            }
+        }
+        return fileUrl;
+    }
```

Comment:
This looks good - I'm wondering if we lose anything here by ignoring the `expires_at` timestamp. Can definitely add it later if someone needs it.